index.js

```diff
@@ -43,8 +43,6 @@ function visitBinding (node) {
   assert.ok(typeof node === 'object' && node && typeof node.type === 'string', 'scope-analyzer: visitBinding: node must be an ast node')
   if (isVariable(node)) {
     registerReference(node)
-  } else if (isShorthandProperty(node)) {
-    registerReference(node)
```

Comment:
It turned out that `isVariable` was enough to find the `value` part of the shorthand property just fine!

Reply:
Ahh right, since the property's .value node will also be walked anyway. That makes sense!